counting_sort/Java/CountingSort.java

```diff
@@ -0,0 +1,43 @@
+/*Counting Sort algorithm- In Counting sort, the frequencies of distinct elements of the array 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpeh1osteg/counting_sort/Java/CountingSort.java
+++ b/tmp/tmpeh1osteg/counting_sort/Java/CountingSort.java
@@ -1,4 +1,4 @@
-/*Counting Sort algorithm- In Counting sort, the frequencies of distinct elements of the array 
+/*Counting Sort algorithm- In Counting sort, the frequencies of distinct elements of the array
 to be sorted is counted and stored in an auxiliary array, by mapping its value as an index of 
 the auxiliary array.
 Time Complexity- The below algorithm takes time O(n+m)
```